client/signup/steps/store/index.jsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'lib/analytics';
+import SignupActions from 'lib/signup/actions';
+import StepWrapper from 'signup/step-wrapper';
+
+import LoggedOutForm from 'components/logged-out-form';
+import LoggedOutFormFooter from 'components/logged-out-form/footer';
+import LoggedOutFormLinks from 'components/logged-out-form/links';
+import LoggedOutFormLinkItem from 'components/logged-out-form/link-item';
+import FormTextInput from 'components/forms/form-text-input';
+import FormButton from 'components/forms/form-button';
+import FormLabel from 'components/forms/form-label';
+import FormSectionHeading from 'components/forms/form-section-heading';
+
+
+module.exports = React.createClass( {
```

Comment:
Oops, missed that. We should probably patch the `theme-selection` step as well: (I used that as a base for this one, and didn't notice we were using two different syntaxes):

https://github.com/Automattic/wp-calypso/blob/master/client/signup/steps/theme-selection/index.jsx#L15